debezium-core/src/test/java/io/debezium/util/ThreadsTest.java

```diff
@@ -10,13 +10,19 @@
 
 import java.sql.SQLException;
 import java.time.Duration;
+import java.util.concurrent.ThreadFactory;
 import java.util.concurrent.TimeoutException;
 import java.util.concurrent.atomic.AtomicBoolean;
 
 import org.junit.Test;
 
 public class ThreadsTest {
 
+    private ThreadNameContext threadNameContext = new ThreadNameContext(
+            "test-connector",
+            "${debezium}-${connector.name}-${task.id}",
```

Comment:
Let's add ${connector.name} and ${task.id} on top of the default thread name pattern here too.